### PR TITLE
HuggingFace collab with AMD on MI300x

### PR DESCRIPTION
**Pull request by mfuntowicz**

### CODE REVIEW

File: huggingface-amd-mi300.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: "Hugging Face on AMD Instinct MI300 GPU"
+thumbnail: /blog/assets/optimum_amd/amd_hf_logo_fixed.png
+authors:
+- user: fxmarty
+- user: mohitsha
+- user: seungrokj
+  guest: true
```
Comment (julien-c) [inline huggingface-amd-mi300.md:8]:
```suggestion
  guest: true
  org: amd
```

File: assets/hf-amd-mi300/tgi_mi300_vs_mi250.png
Comment (julien-c) [inline assets/hf-amd-mi300/tgi_mi300_vs_mi250.png:1]: the assets you should add them to the documentation-images repo instead

File: _blog.yml [hunk 2]
```diff
@@ -4014,4 +4014,19 @@
   tags:
     - cloud
     - azure
-    - partnership
\ No newline at end of file
+    - partnership
+
+- local: huggingface-amd-mi300
```
Comment (jeffboudier) [inline _blog.yml:4019]: not sure having 'huggingface' in the URL stub is helpful, maybe 'amd-mi300' is shorter and sweeter?

File: _blog.yml [hunk 3]
```diff
@@ -4014,4 +4014,19 @@
...
+  title: "Hugging Face on AMD Instinct MI300 GPU"
+  thumbnail: /blog/assets/optimum_amd/amd_hf_logo_fixed.png
+  author: huggingface-team
```
Comment (jeffboudier) [inline _blog.yml:4022]: Is this going to create a broken link? Let's add the actual authors I think

**Review by jeffboudier — APPROVED** [review]
Super informative post!

**Pull request merged**